docs/samples/deployments/data-factory/presidio-data-factory-template-gallery-databricks.md

```diff
@@ -0,0 +1,63 @@
+
+**Note that** this solution is capabale of transforming large datasets. For smaller, text based input you may want to work with the Data Anonymization with Presidio (HTTP) template which offers an easier deployment for Presidio.
+
+The sample deploys the following Azure Services:
+
+* Azure Storage - The target storage account where data will be persisted.
+* Azure Databricks - Host presidio to anonymize the data.
+
+Additionaly you should already have an instance of Azure Data Factory which host and orchestrate the transformation pipeline and a storage account which holds the source files.
+
+## About this Solution Template
+
+This template gets the files from your source file-based store. It then anonymizes the content and uploads each of them to the destination store.
+
+The template contains three activities:
+
+* **AnonymizeSource** runs the presidio notebook on the input file to create an output folder with csv parts.
+* **MergeAnonymizedToTarget** merges the csv parts from databricks output folder to a single csv on the target storage container.
+* **DeleteAnonymized** deletes the temporary output folder.
+
+The template defines four parameters:
+
+* **SourceStore_Location** is the container name of your source store where you want to move files from (STORAGE_CONTAINER_NAME).
+* **DestinationStore_Name** is the container name in the target storage account which is provisioned by the ARM template.
+* **SourceFile_Name** is the name of the input file.
+* **TextColumn_Name** is the name of the column to be anonymized.
+
+## How to use this Solution Template
+
+To use this template you should first setup the required infrastructure for the sample to run, then setup the template in Azure Data Factory.
+
+### Setup Presidio
+
+Provision and setup the datbricks cluster by following the steps in [presidio-spark sample](../spark/index.md#Azure-Databricks).
+Note the output token.
+
+### Setup Azure Data Factory
+
+1. Go to the Data anonymization with Presidio (Databricks) template. Select the Anonymized_CSV connection (Azure Storage) and select "New" from the drop down menu.
+![ADF-Template-Load](images/data-anonymization-databricks-01.png)
+
+2. Name the service "PresidioStorage" and select the storage account that was created in the previous steps from your subscription. Note that Target source was also select as the sample uses the same storage account for both source and target.
+![ADF-Storage-Link](images/data-anonymization-databricks-02.png)
+
+3. Select the Anonymize Source connection (Databricks) and select "New" from the drop down menu.
+![ADF-Databricks-Load](images/data-anonymization-databricks-03.png)
+
+4. Name the service "PresidioDatabricks" and select the Azure Databricks workspace that was created in the previous steps from your subscription. Follow through the steps to input the authentication token which was generated in the previous step and selecting presidio_cluster to run the job.
```

Comment:
add a link to the step where the authentication token to databricks was generated

Reply:
done